docs/source/models.rst

```diff
@@ -443,14 +441,14 @@ MLflow includes the utility function ``build_and_push_container`` to perform thi
 container for all MLflow models. Model webservers deployed using the :py:mod:`mlflow.sagemaker`
 module will accept the following data formats as input, depending on the deployment flavor:
 
-  * `mlflow.pyfunc.FLAVOR_NAME`: For this deployment flavor, The endpoint accepts the same formats
+  * ``python_function``: For this deployment flavor, The endpoint accepts the same formats
     as the pyfunc server. These formats are described in the
     :ref:`pyfunc deployment documentation <pyfunc_deployment>`.
 
-  * `mlflow.mleap.FLAVOR_NAME`: For this deployment flavor, the endpoint ONLY accepts
-    JSON-serialized Pandas DataFrames in the `split` orientation. For example,
-    `data = pandas_df.to_json(orient='split')`. This format is specified using a `Content-Type`
-    request header value of `application/json`.
+  * ``mleap``: For this deployment flavor, the endpoint ONLY accepts
+    JSON-serialized Pandas DataFrames in the ``split`` orientation. For example,
```

Comment:
For this deployment flavor, the endpoint ONLY accepts -> For this deployment flavor, the endpoint accepts `only`